test/ApplicationTest.php

```diff
@@ -22,6 +23,12 @@ class ApplicationTest extends TestCase
 {
     use ProphecyTrait;
 
+    /** @var ContainerInterface|ObjectProphecy */
```

Comment:
We can use native types in here, if needed, otherwise all good, so this is just a nit

Reply:
Done.